pkg/repo/fs.go

```diff
@@ -98,6 +98,11 @@ func (fsr *FsRepo) Init() error {
 	// TODO this should be a part of the config.
 	telemetry.SetupFromEnvs()
 
+	// never fail here as this isn't critical to node start up.
+	if err := fsr.WriteInstanceID(GenerateInstanceID()); err != nil {
+		log.Trace().Err(err).Msgf("failed to write instanceID")
```

Comment:
we should log as warning right?

Reply:
We should if we want to warn users we were unable to generate an instanceID, but I don't see any action they can take to right the problem, what do you think?

imo the instanceID is best effort, if for some reason we can't write it (which should be _really_ rare) then I don't see a point warning them as it doesn't affect the operation of bacalhau.